README.md

```diff
@@ -21,20 +22,44 @@ docker-compose up -d      # Starts an instance of the openrecord api and a mysql
 
 ## Configuration
 
-Configuration is managed using [node-config](https://github.com/lorenwest/node-config). *All* configuration options should go in the _config/_ directory.
+Configuration is managed using [node-config](https://github.com/lorenwest/node-config). _All_ configuration options should go in the _config/_ directory.
 
-Configs are inherited from _default.yml_ initially. When NODE_ENV is set, a file matching _[NODE_ENV].yml_ will be loaded (i.e. staging and production).
+Configs are inherited from _default.yml_ initially. When NODE_ENV is set, a file matching _NODE_ENV.yml_ will be loaded (i.e. staging and production).
 
 To override any configuration, add a file called _local.yml_ (git-ignored) or map an environment variable to a configuration option in _custom-environment-variables.yml_.
 
 Example configuration setup for `API_PORT=1234 NODE_ENV=staging npm start`: _default.yml_ -> _staging.yml_ -> _local.yml_ -> _local-staging.yml_ -> Environment Variables (_API_PORT_)
 
+## End-to-End Tests with Puppeteer
+
+Start postgres and prisma.
+`docker-compose up -d prisma`
+Deploy the turntable prisma schema.
+`docker-compose up turntable-deploy`
+
+#### Local Chrome
+
+Start vinyl locally.
+`npm start`
```

Comment:
Unfortunately, Markdown isn't preserving these newlines. I thik you have to use the \`\`\` to ensure the code snippets go on their own line.